### PR TITLE
Tailwind setup

### PR DESCRIPTION
**Pull request by benjaminbiome**
Yo,

Man i am tired - been coding all day and forgot about lunch.

Added all the components etc that we need + change some text.

I have connected to the notion API to render our notion content - I am not sure if I did this correctly as should be on the server side.

I will send you keys over slack. 

I'd like to know if you can make 2 API calls within the getStaticProps() function?

The article page is currently just rendering 1 page in notion. I need to make this dynamic. 

Cheers

### CODE REVIEW

File: pages/api/hello.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
-// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+const { Client } = require('@notionhq/client');
```
Comment (peterbiome) [inline pages/api/hello.js:1]: i suppose we'll rename the file name in future ;)

File: pages/article.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        <WaitingList/>
+        <Footer/>
+        </div>
+    )
+}
+
+export async function getStaticProps() {
+    const notion = new Client({ auth: process.env.NOTION_API_KEY });
+    const response = await notion.blocks.children.list({
+        block_id: "3797d457d5c343ff9487a6fc5efdc7c3",
```
Comment (peterbiome) [inline pages/article.js:144]: is this the `single article` id?  that will be added as the dynamic route param, right?

**Comment by peterbiome** [comment]
I've added the secrets to Vercel so the deploys should not fail now

File: pages/article.js (hunk 2 above)
Comment (benjaminbiome) [reply pages/article.js:144]: Yeh so at the moment i have just hardcoded an Id, the aim is to to make this dynamic. I figures out how to pass it through to page as you suggested, but then when i referenced it here it didn't work.

I'll do some more research tomorrow.

**Pull request merged**